Protein/examples/0.11-RELEASE/nextprot-jsonld.json

```diff
@@ -0,0 +1,36 @@
+{
+    "@context": "http://schema.org",
+    "@type": "DataRecord",
+    "@id": "https://www.nextprot.org/entry/NX_P52701",
+    "includedInDataset": "ftp://ftp.nextprot.org/pub/current_release/xml/nextprot_all.xml.gz",
+    "citation": {
```

Comment:
Omit this property if there isn't a value for it.

You may also want to add a `citation` property into your dataset markup